mipengine/node/sql_injection_guard.py

```diff
@@ -0,0 +1,135 @@
+import inspect
+import re
+from functools import wraps
+from inspect import getfullargspec
+
+
+def get_arg_value_from_args_and_kwargs(func, param_name, args, kwargs):
+    arglist = getfullargspec(func)
+    if kwargs.get(param_name):
+        return kwargs.get(param_name)
+    else:
+        param_index = arglist.args.index(param_name)
+        if param_index < len(args):
+            return args[param_index]
+
+    return None
```

Comment:
Nit (kind of):
It seems that only passed values are validated, not default ones. Why not validate everything? Developers also make mistakes.